metric-anomaly-data-model/src/main/proto/org/hypertrace/alert/engine/eventcondition/config/service/v1/metric_anomaly_event_condition.proto

```diff
@@ -87,6 +89,10 @@ message StaticThresholdCondition {
   Severity severity = 4;
 }
 
+message DynamicThresholdCondition {
+  string baseline_duration = 2;  // iso duration string
```

Comment:
Currently, this supports defining `calculate using historical data of past X day`. As @rish691 mentioned, the baseline function for bounds will be exposed as a follow-up.

@rish691 Do we have a task ticket for that? If, not, can we add one as part of the phase2 item?

Reply:
https://github.com/hypertrace/hypertrace-alert-engine/issues/53 please add details on this on what all is required to expose